packages/rabbitmq/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { CSL, cslToCore } from '@cardano-sdk/core';
+
+export const TX_SUBMISSION_QUEUE = 'cardano-tx-submit';
+
+export const txBodyToId = (txBody: Buffer | Uint8Array, dummy?: boolean) => {
+  if (dummy) {
+    const buffer = txBody instanceof Buffer ? txBody : Buffer.from(txBody);
+
+    return buffer.toString('hex');
+  }
+
+  const buffer = txBody instanceof Buffer ? txBody : Buffer.from(txBody);
+  const txDecoded = CSL.Transaction.from_bytes(buffer);
```

Comment:
I'm wondering if we should wrap this under a new util that would convert `Cardano.util.HexBlob` into `Cardano.NewTxAlonzo`. I think we should aim to not use CSL directly. What do you think @rhyslbw

Reply:
Agree. A util here makes sense

Reply:
`Cardano.util.deserializeTx`?